interfaces/repo_test.go

```diff
@@ -251,6 +247,7 @@ func (s *RepositorySuite) TestAddPlugFailsWithUnknownInterface(c *C) {
 	c.Assert(s.emptyRepo.AllPlugs(""), HasLen, 0)
 }
 
+/*
```

Comment:
Can you explain why this is being disabled?

Reply:
Actually I meant to remove this. Removed now.